inputs_demand.md

```diff
@@ -0,0 +1,172 @@
+| Retirement rules | For vintaged technologies, GCAM requires the user to specify the lifetime, and the parameters required for phased and profit-based shutdown. | Assumption |  | Specified by technology and year | Years (for lifetime), unitless for others |
+| Logit exponents | GCAM requires the user to specify the logit exponents that determine the substitutability between technologies. | Assumption |  | Specified by sector and subsector | N/A |
+| Share weight interpolation rules | These rules dictate how share weights (GCAM's calibration parameter) are specified in future years. | Assumption |  | Specified by sector and subsector | N/A |
+| Fuel preference elasticity | Elasticity dictating how share weights change with GDP per capita | Assumption | Specified by technology and year | unitless |
+
+
+Table 1: External inputs used for demand of energy<sup>[1](#table_footnote1)</sup>
+
+Note that for the Shared Socioeconomic Pathways (SSPs), different inputs are used for some variables. See [SSPs](ssp.html) for more information.
+
+#### Data
+
+Throughout GCAM, the number in the name of assumption file indicates to which sector the file applies. Files with `A32` in the name are assumptions for industry, `A321` indicates cement, and `A322` indicates fertilizer. Files with `A44` in the name are assumptions for buildings. Files with `A54` in the name are assumptions for transportation. 
+
+##### Historical demand for energy
+
+GCAM uses IEA energy balances as a source for historical energy supply and demand. IEA data are proprietary and thus are not provided in the GCAM data repository. Instead, we provide all of the `R` code used to process the IEA data so that the user can replicate the processing _if_ they purchase the IEA data. In addition, we provide aggregated data after it has undergone processing so that GCAM input files can be created and used by the user community. 
+
+##### Elasticities of demand
+
+Price elasticity of demand is specified in [A32.demand.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A32.demand.csv), [A54.demand.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.demand.csv) .
+
+##### Cost
+
+Costs are specified in [A32.globaltech_cost.csv](https://github.com/JGCRI/gcam-core/tree/master/input/gcamdata/inst/extdata/energy/A32.globaltech_cost.csv), [A321.globaltech_cost.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A321.globaltech_cost.csv), [A44.cost_efficiency.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A44.cost_efficiency.csv)
+
+##### Default efficiencies
+Efficiencies are specified in [A32.globaltech_eff.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A32.globaltech_eff.csv), [A44.cost_efficiency.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A44.cost_efficiency.csv)
+.
```

Comment:
Just in case, in the buildings sector the efficiencies are extended to the ROW using the multipliers in this file: A44.tech_eff_mult_RG3.csv[https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A44.tech_eff_mult_RG3.csv]